lib/game.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# class Game checks for the rows,columns and diagonals to check if anyone has won or else
+# tracks the count of turns and returns a draw if the bosrd is full before anyone wins.
+class Game
+  def initialize
+    @count = 0
+  end
+
+  def status(row_col_diagonals, sign)
+    @count += 1
+    return "#{sign} won" if row_col_diagonals.any? { |element| element.count(sign) == 3 }
+    return 'draw' if @count >= 9
+
+    'continue'
+  end
+end
+
+# test
+
+arr = [[1, 2, 3]]
```

Comment:
If you truly think you need these lines of code, try adding it to a new file in bin folder called `test.rb` etc.

Question, do you need help arranging your project files and accessing classes across the whole project?

Reply:
Good morning mcrd25 this is a feature branch, it is not supposed to play the game, it is only to provide a way to get the status of the game and test whether or not anyone won!